test/lib/jdk/test/lib/process/OutputAnalyzer.java

```diff
@@ -862,4 +864,11 @@ public void shouldContainMultiLinePattern(String... needles) {
         shouldContainMultiLinePattern(needles, true);
     }
 
+    /**
+     * Assert that we did not crash with a hard VM error (generating an hs_err_pidXXX.log)
+     */
+    public void shouldNotHaveFatalError() {
```

Comment:
Looking at the usages, I wonder if this should be `checkFatalError(boolean)` and, similarly, for the exit value `checkExitValue(int value)` ?

Reply:
Are you suggesting to rename the existing methods? I don't see `checkExistValue(int value)` in `OutputAnalyzer`. The existing method naming pattern for the assertions is always `should(Not)XXX` though, so that's what I followed here.